### DISCUSSION TITLE
My PW2 seems to have a reported ceiling of around 4.5 kWh when exporting to the grid.

### DISCUSSION THREAD

**Discussion by Stephen2615** [jasonacox/Powerwall-Dashboard#566] [General]
Greetings,

For what seems like an eternity, I have been trying to figure out why my electricity provider's data is different to my Powerwall's reported data. My load/home data is consistently higher than what it is in reality. I graphed my electricity providers five minute intervals and during peak production, I am exporting around 5.7 kWh when making around 7.75 kWh. The PW rarely reports anything higher than around 4.5 kWh. Over the day, that difference equates to the higher load/home information that is reported on the Dashboard. I work on the assumption that load is what is left over from solar created - grid export - battery = load. If I produce say 7.75 kWh and my export is 5.7 kWh with a fully charged battery, load is  about 2.05 kWh which is what I think is right.  The PW seems to work on the 7.75 kWh - 4.5 kWh = 3.25 kWh which over the day amounts to the error I think is there. 

I am just wondering if this is a bug with PW2 grid export stats or something different. Any comments would be very welcome. 

TIA!!

Stephen

**Comment by jasonacox** [comment]
@Stephen2615 - Just so I am clear, the difference is between Tesla and your electric provider data?  In other words, are you seeing a significant difference between what the Tesla App reports and what you see in Powerwall Dashboard?

**Comment by Stephen2615** [comment]
@jasonacox 

Hi Jason, well it seems that I have discovered the anomaly and I really should have done a search before posting. The PW 2 has an official export limit of 5 kWh. Mine rarely gets past 4.4 kWh. Yesterday,  my peak export was almost 7.5 kWh between 12:00 and 13:00 but it does not change for daylight saving. So the time is 13:00 to 14:00. The dashboard shows around 4.4 kWh at that time. The limit set by the PW2 obviously doesn't impact what actually goes to the grid. It does, however, affect the reporting coming from the battery itself. So if my assumption that load is what is left over from solar production minus grid export minus battery = load, the anomaly is why my load is much higher than it is in reality. I don't know how your code can report the true value of load if this is what is happening. 

Just a bit crazy if you ask me. 

Cheers

Stephen

![Actew-agl-29Dec2024](https://github.com/user-attachments/assets/4352680f-b669-42e1-ad9d-b615a4158924)
![tesla-to-grid-29Dec2024](https://github.com/user-attachments/assets/077cdcaf-96f9-479d-a868-92ede021e1f1)

**Reply by Stephen2615** [reply]
Another piece of the pie that might be relevant. I have three phase power but the Tesla PW2 is only on one phase. Our local electricity provider has a limit of 5 kW on single phase but 15 kW on three phase. I wonder if the Tesla doesn't know about the other two phases and restricts its grid export to 5 kW and the rest of the power gets past it down the other two phases to the grid which might explain the up to about 3 kW difference between the actual grid export and Tesla's grid export and, therefore, the unusually high load/home reading.
